internal/serve/httphandler/payments_handler.go

```diff
@@ -0,0 +1,34 @@
+package httphandler
+
+import (
+	"net/http"
+	"wallet-backend/internal/data"
+	"wallet-backend/internal/serve/httperror"
+
+	"github.com/stellar/go/support/http/httpdecode"
+)
+
+type PaymentsHandler struct {
+	*data.Models
+}
+
+type PaymentsSubscribeRequest struct {
+	Address string `json:"address"`
+}
+
+func (h PaymentsHandler) SubscribeAddress(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	var reqBody PaymentsSubscribeRequest
+	err := httpdecode.DecodeJSON(r, &reqBody)
+	if err != nil {
+		httperror.BadRequest("Invalid request body").Render(w)
+		return
+	}
+
+	err = h.Models.Payments.SubscribeAddress(ctx, reqBody.Address)
+	if err != nil {
```

Comment:
I think that's a good idea! 
I was considering how we should implement Sentry though. I think it might be worth implementing an error tracking interface so that contributors could easily implement other providers.
I've created a [task](https://app.asana.com/0/1203779254475146/1207245825676510/f) for it and left a TODO in the code: [1c32a92](https://github.com/stellar/wallet-backend/pull/6/commits/1c32a922ff4286bf8126a351a7ac43e0d57b856e)

Reply:
Btw, I see that we are currently sharing Asana links over PR's (it's even in the template). Should we stop doing so and remove it from the template?
cc: @howardtw @mtwtfss

Reply:
Or move the all wallet-backend tasks to Github issues?